magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/source_dict.json

```diff
@@ -0,0 +1,125 @@
+{"TCrB":[239.8756667, 25.91667, "NaN"],
```

Comment:
it would be useful to sort this list somehow: by source name, by RA, or having sources with RA/Dec specified before the ones with NaNs. Now it looks like the sequence is rather random